oscarapi/apps/basket/abstract_models.py

```diff
@@ -0,0 +1,101 @@
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from oscar.apps.basket.abstract_models import AbstractBasket as _AbstractBasket, AbstractLine as _AbstractLine
+from oscar.apps.basket.managers import OpenBasketManager, SavedBasketManager
+from oscar.core.utils import get_default_currency
+
+from oscarapi.apps.basket.managers import EditableBasketManager
+from oscarapi.apps.basket.utils import get_basket
+
+
+__all__ = ('AbstractBasket', 'AbstractLine')
+
+
+class AbstractBasket(_AbstractBasket):
```

Comment:
Could you explain why you need to override `Basket` and `Line`? Since this is to be a generic API package, I would have thought we would avoid overriding core apps like basket.